package.json

```diff
@@ -0,0 +1,69 @@
+{
+    "name": "mv-bookstore",
+    "version": "0.1.0",
+    "homepage": "https://Ol-create.github.io/book-store",
+    "private": true,
+    "dependencies": {
+        "@fortawesome/fontawesome-svg-core": "^6.1.1",
+        "@fortawesome/free-regular-svg-icons": "^6.1.1",
+        "@fortawesome/free-solid-svg-icons": "^6.1.1",
+        "@fortawesome/react-fontawesome": "^0.1.18",
+        "@testing-library/jest-dom": "^5.16.4",
+        "@testing-library/react": "^13.1.1",
+        "@testing-library/user-event": "^13.5.0",
+        "prop-types": "^15.8.1",
+        "react": "^18.1.0",
+        "react-dom": "^18.1.0",
+        "react-redux": "^8.0.2",
+        "react-router-dom": "^6.3.0",
+        "react-scripts": "5.0.1",
+        "redux": "^4.2.0",
+        "redux-thunk": "^2.4.1",
```

Comment:
- [ ] In this project, you are expected to use the modern way of working with redux which is by using the [REDUX TOOLKIT](https://redux-toolkit.js.org/introduction/getting-started) This is because it comes with functions that help you manage the state and configure the store in a very easy way and on the top of that, it is easy to get started with it as its documentation is really contentful 😃 👍  so please use it here in your project too 🏃